php/PostTypes/BlockPost.php

```diff
@@ -402,7 +402,7 @@ public function save_draft_button() {
 	public function render_properties_meta_box() {
 		$post  = get_post();
 		$block = new Block( $post->ID );
-		$icons = genesis_custom_blocks()->get_icons();
+		$icons = [];
```

Comment:
Most of this file will be deleted when this editor redesign is done. It mainly renders the editor and handles its data.